### ISSUE TITLE
"Fatal error" - csgosl.vbs is not starting anymore still occurs

### ISSUE THREAD

**Issue by exxis** [lenosisnickerboa/csgosl#71]
i´am running the current version but still get the same error like @Rasko1337 from Post #69 got.

https://i.gyazo.com/3622dcaed002141d7bd43521833aa5dd.png

**Comment by lenosisnickerboa** [comment]
Yes, it is the same kind of error but not the exact same location. I must
have missed this particular case. A new fix is coming and thanks for
reporting it.

Den 8 mars 2017 1:19 fm skrev "exxis" <notifications@github.com>:

i´am running the current version but still get the same error like
@Rasko1337 <https://github.com/Rasko1337> from Post #69
<https://github.com/lenosisnickerboa/csgosl/issues/69> got.

https://i.gyazo.com/3622dcaed002141d7bd43521833aa5dd.png

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/lenosisnickerboa/csgosl/issues/71>, or mute the thread

**Comment by lenosisnickerboa** [comment]
Well, I just had a look and the error you get should be fixed in the 1.3.1 version. Can you verify that you are running the 1.3.1 version?

**Comment by exxis** [comment]
Yes, I've read the other thread, then i thought good it's fixed in 1.3.1, then i checked my Version and i already had it. 
Maybe the errror is a little different, as map group I've selected a collection from the Workshop and as startmap a map from the same collection. 
At the Server start all maps were downloaded and working then i restarted csgosl and the error appears. I've then deleted the Workshop folder and let the server download the maps again. 

I've now to check if the fatal error appears again.
